qiskit_experiments/library/tomography/tomography_analysis.py

```diff
@@ -71,7 +71,7 @@ def _default_options(cls) -> Options:
                 supply a custom fitter function. See the `Fitter Functions` section for
                 additional information.
             fitter_options (dict): Any addition kwarg options to be supplied to the fitter
-                function. For documentation of available kargs refer to the fitter function
```

Comment:
`kargs` to `kwargs` which I assumed stood for “keyword arguments”. My understanding is that the standard in Python is to name these `kwargs`.

Reply:
What do you mean by naming these? The google docstring standard doesn't necessarily expand kwargs, see https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html